x-pack/plugins/spaces/public/plugin.tsx

```diff
@@ -133,7 +143,7 @@ export class SpacesPlugin implements Plugin<SpacesPluginSetup, SpacesPluginStart
 
     registerAnalyticsContext(core.analytics, this.spacesManager.onActiveSpaceChange$);
 
-    return { hasOnlyDefaultSpace };
+    return { hasOnlyDefaultSpace, isSolutionViewEnabled: allowSolutionVisibility };
```

Comment:
__note:__ since `isSolutionViewEnabled` is a part of our public API now, can you please add a test or two that make sure it respects `isCloudEnabled`, `config.allowSolutionVisibility`, and `isServerless` (no need to account for `forceSolutionVisibility` since it's a throw-away work unless you want to)?

Reply:
Totally agree, not sure how I missed that. Addressed in https://github.com/elastic/kibana/pull/191743/commits/a289827faf478c70d34d38b761903809b34496cf